apps/docs/components/toggleButton/toggleButton.css

```diff
@@ -0,0 +1,41 @@
+.hd-toggleButton {
+    --toggle-button-selected-background: var(--hd-color-neutral-surface-weak);
+    --toggle-button-hover-background: var(--hd-color-neutral-surface-weak);
+    --toggle-button-color: var(--hd-color-neutral-icon);
+
+    background-color: transparent;
+    color: var(--toggle-button-color);
+    border-width: 0;
+    border-radius: var(--hd-space-05);
+    display: flex;
+    gap: var(--hd-space-2);
+    justify-content: center;
+    align-items: center;
+    font-size: 1rem;
+
+    @media screen and (width >= 37.5rem) {
+        width: var(--hd-space-4);
+        aspect-ratio: 1/1;
+
+        &:hover {
```

Comment:
does this actually still work? Just realized that you are doing sass in a css file.

Reply:
it works well it's the magic of Nesting css 

https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector

Reply:
I guess it's supported now. I just thought that we weren't using this? or is it different int he docs than components?